src/app/use-case-list/use-case-list.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+import UseCases from '../../../mocks/data.json';
```

Comment:
We will use data.json to mock the api, but the idea is to call the http services anyway.
So, instead importing the json, it's the backend.service that has to be injected and a new method there will return an Observable.

Reply:
I see your point and I agree